src/plumbing/fnk/impl.clj

```diff
@@ -37,6 +37,16 @@
   [k]
   (symbol (name k)))
 
+(defn qualified-k->sym
+  "Make a keyword into a symbol, preserving namespace"
+  [k]
+  (symbol (namespace k) (name k)))
```

Comment:
This can just be called keyword->symbol?  or k->sym.  It also works on unqualified.